ShopifySharp/Services/ShopifyService.cs

```diff
@@ -14,35 +14,39 @@ namespace ShopifySharp
 {
```

Comment:
This file currently has too many unrelated changes for me to merge, particularly the debug logger printing.